plugins/additional/additional.cpp

```diff
@@ -370,14 +378,18 @@ class RichTextCtrlComponent : public ComponentBase
 	}
 
 	ticpp::Element* ExportToXrc(IObject* obj) override {
-		ObjectToXrcFilter xrc(obj, _("unknown"), obj->GetPropertyAsString(_("name")));
-		//xrc.AddWindowProperties();
+		ObjectToXrcFilter xrc(obj, _("wxRichTextCtrl"), obj->GetPropertyAsString(_("name")));
+		xrc.AddWindowProperties();
+		if(!obj->GetPropertyAsString(_("value")).IsEmpty())
+		{
+			xrc.AddProperty(_("value"), _("value"), XRC_TYPE_TEXT);
+		}
 		return xrc.GetXrcObject();
 	}
 
 	ticpp::Element* ImportFromXrc(ticpp::Element* xrcObj) override {
 		XrcToXfbFilter filter(xrcObj, _("wxRichTextCtrl"));
-		filter.AddWindowProperties();
```

Comment:
That looks like a mistake, the standard properties should also be handled

Reply:
Definitely mistake, fixed. Also updated corresponding to comments above.